package.json

```diff
@@ -30,7 +32,7 @@
     "build:test": "webpack --config webpack.test.config.js",
     "prestart": "npm run build:dev",
     "start": "grunt start-server-open",
-    "prepublish": "grunt prep-release",
+    "prepublish": "grunt prep-release && webdriver-manager update",
```

Comment:
How about in postinstall? Or before we actually run our tests (since that's the only time we care)?